UnityProject/Assets/Resources/Icons/Obj/supermatter.meta

```diff
@@ -1,10 +0,0 @@
-fileFormatVersion: 2
-guid: addd0b3c9fc9249ab95dfb2e249c7b92
-folderAsset: yes
-timeCreated: 1511877770
-licenseType: Free
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
```

Comment:
this .meta is unrelated and shouldn't be in the PR, same with telescience.meta, walls.meta, etc